docs/generate_command_docs.py

```diff
@@ -102,7 +102,8 @@ def create_file_if_not_exist(filepath: str, cmd: str) -> bool:
 def generate_help_command_and_output(cmd: str) -> typing.Tuple[str, str]:
     help_cmd = ["pebble", "help"] if cmd == "help" else ["pebble", cmd, "--help"]
     help_cmd_str = " ".join(help_cmd)
-    help_cmd_output = get_command_help_output(help_cmd_str).strip()
+    go_run_cmd = " ".join(["go", "run", "../cmd/pebble"] + help_cmd[1:])
+    help_cmd_output = get_command_help_output(go_run_cmd).strip()
```

Comment:
It's weird to create `help_cmd` with "pebble" at the start, only to remove it and replace it here. Can we do something like:

```python
args = ["help"] if cmd == "help" else [cmd, "--help"]
help_cmd_str = " ".join(["pebble"] + args)
go_run_cmd = " ".join(["go", "run", "../cmd/pebble"] + args)
```